documentation/endpoints/activity.md

```diff
@@ -17,7 +17,7 @@ GET /api/v2/activity
   - For pagination, specify page of activity to return in the response.
   - e.g. `/activity?page=2`
 - **include** _(string)_
-  - Include additional related records in the response: `user`
+  - Include additional related records in the response: `user`, `posts`
```

Comment:
Nitpicky - But can multiple values be pass in this param? If so, can we add an example and specify that to add multiple include params they need to be comma-separated?

Reply:
yes great idea!